### PR TITLE
Fixes for two crashes

### PR DESCRIPTION
**Pull request by janbrothanek**
Fixes for two crashes which we found at Comprimato while integrating NTV2.

### CODE REVIEW

**Comment by janbrothanek** [comment]
@mrbillaja I see I reported the related bugs as #4725 and #4609 in sdksupport.aja.com, and we were asked to reproduce them with newer versions where there were potential fixes. I'm sorry I forgot about that. We always apply the two patches on top of the SDK, just now when the SDK goes to Github, we thought it would be nicer to make a pull request. I'll try to reproduce it sometime soon.

File: ajaanc/src/ancillarylist.cpp [hunk 1]
```diff
@@ -986,6 +986,9 @@ AJAStatus AJAAncillaryList::AddFromDeviceAncBuffer (const NTV2Buffer & inAncBuff
 	const uint32_t	origPktCount	(outPackets.CountAncillaryData());
 	AJAStatus		result			(AJA_STATUS_SUCCESS);
 
+	if (inAncBuffer.GetByteCount() == 0)
```
Comment (mrbillaja) [inline ajaanc/src/ancillarylist.cpp:989]: @janbrothanek This check isn't necessary. If `inAncBuffer` is empty/null, `BufferHasGUMPData` will return false (without crashing, it first checks for null/empty), and in the RTP 'else' case, `ancBuffer` will be empty/null too, so the `while` loop cannot be entered. The function will log zero packets added, and return `AJA_STATUS_SUCCESS`. I believe this is tested here: https://github.com/aja-video/libajantv2/blob/c6c784a9bd8fd8f27bc3421aec9ed52dd01a7dcc/ajaanc/test/main.cpp#L2352

File: ajantv2/src/ntv2driverinterface.cpp [hunk 2]
```diff
@@ -591,7 +591,10 @@ bool CNTV2DriverInterface::DriverGetBitFileInformation (BITFILE_INFO_STRUCT & bi
 	if (!::NTV2DeviceHasSPIFlash(_boardID))
 		return false;
 
-	ParseFlashHeader(bitFileInfo);
+	if (!ParseFlashHeader(bitFileInfo)) {
```
Comment (mrbillaja) [inline ajantv2/src/ntv2driverinterface.cpp:594]: @janbrothanek Could you remove the braces to match the existing coding style in that function (e.g. as in line 591-592)?

Comment (paulh-aja) [reply ajantv2/src/ntv2driverinterface.cpp:594]: @janbrothanek Please open a separate PR for this change.

As @mrbillaja says that the change to "ajaanc/src/ancillarylist.cpp" is not needed I am going to close this PR.

**Pull request closed without merging**